### PR TITLE
feat(disease-outbreaks): Sprint 2 — content-age pilot (synthetic tagging + 9d budget)

### PR DESCRIPTION
**Pull request by koala73**
**Stacked on PR #3596** (Sprint 1 — content-age probe infra). Merge #3596 first; this PR rebases onto main automatically once that lands.

Implements Sprint 2 of the 2026-05-04 health-readiness plan. Migrates disease-outbreaks as the proof-of-concept content-age consumer. Pilot `maxContentAgeMin = 9 days` chosen so the **2026-05-04 incident** (cache held 50 outbreaks all 11+ days old, /api/health reported OK, map went empty) would have correctly tripped `STALE_CONTENT`.

### 1. Synthetic-timestamp tagging across all 3 sources

| Parser | Existing fallback | Migration |
|---|---|---|
| `fetchWhoDonApi` (line ~117) | `Date.now()` when `PublicationDateAndTime` missing | Carry `_originalPublishedMs` (parsed ms or null) + `_publishedAtIsSynthetic` (boolean). `publishedMs` keeps `Date.now()` fallback for UI consumer compat. |
| `fetchRssItems` (line ~150, both CDC and Outbreak News Today) | `Date.now()` when `pubDate` missing | Same pattern |
| `fetchThinkGlobalHealth` (line ~211) | line-198 filter rejects undated already | Always `_publishedAtIsSynthetic: false`, `_originalPublishedMs: publishedMs` (uniform shape across sources) |

`mapItem` (line ~244) carries the helpers through to the output shape so `contentMeta` can read them at runSeed time.

### 2. runSeed contract opts

- **`contentMeta`**: excludes `_publishedAtIsSynthetic` items + 1h clock-skew tolerance + null when `validCount === 0`. Synthetic-timestamp items would otherwise falsely report content as fresh and mask real upstream silence.
- **`maxContentAgeMin: 9 * 24 * 60 = 12960`** (9 days). Tighter would page on normal WHO/CDC quiet weeks; looser would have missed the 11d incident.
- **`publishTransform`**: strips `_publishedAtIsSynthetic` + `_originalPublishedMs` from every item BEFORE atomicPublish. Per Sprint 1's ordering contract, contentMeta has already run on the raw data by this point — safe to drop. The strip ensures helpers never reach the canonical key, /api/bootstrap response, list-disease-outbreaks RPC response, or the proto-generated type.

## Tests (`tests/disease-outbreaks-seed.test.mjs` — NEW)

16 cases split by **layer** per Codex round 4-5 review (pre-publish in-memory CARRIES helpers; post-strip canonical is helper-free):

| Layer | Cases |
|---|---|
| Pre-publish (parser) | WHO/RSS without date → synthetic; WHO/RSS with date → non-synthetic; TGH always non-synthetic |
| `contentMeta` behavior | All-synthetic → null; mixed → synthetic doesn't win newest; newest/oldest selection; future-dated >1h excluded; near-future ≤1h accepted |
| `publishTransform` strip | Both fields stripped; `publishedAt` remains non-null; empty/missing handled |
| End-to-end | Combined-regex assertion: published JSON contains NEITHER helper name |
| Pilot threshold | 11d-old items DO trip; 5d-old items DO NOT (anti-drift) |

**Test totals:** 95/95 pass across seed-envelope, seed-contract, seed-utils, content-age, health-content-age, and disease-outbreaks-seed suites.

## Net diff

```
2 files changed, 449 insertions(+), 10 deletions(-)
```

## Verification (post-deploy)

After Railway redeploy of `seed-bundle-health`:

- [ ] `/api/health.diseaseOutbreaks` reports `contentAgeMin` and `maxContentAgeMin: 12960`
- [ ] With current cache (50 items, newest ~11d old per 2026-05-04 incident), `/api/health.diseaseOutbreaks.status === 'STALE_CONTENT'`
- [ ] Redis canonical helper-free:
  ```bash
  redis-cli GET health:disease-outbreaks:v1 | grep -cE '_publishedAtIsSynthetic|_originalPublishedMs'
  # Expected: 0
  ```
- [ ] Same combined-regex check on `/api/bootstrap?keys=diseaseOutbreaks` response body returns 0
- [ ] After TGH fix from #3593 lands daily-fresh items, STALE_CONTENT clears naturally as `newestItemAt` shifts to within 9 days

## What's next

Sprint 3 — sparse seeders (climate news, IEA OPEC, news-digest, economic stress).
Sprint 4 — annual-data seeders (WB resilience indicators).

### CODE REVIEW

File: tests/disease-outbreaks-seed.test.mjs [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+    _originalPublishedMs: publishedMs,
+    _publishedAtIsSynthetic: false,
+  };
+}
+
+// ── Pre-publish (in-memory) layer ────────────────────────────────────────
+
+test('WHO record without PublicationDateAndTime → in-memory item is tagged synthetic', () => {
+  const NOW = 1700000000000;
+  const inMemory = whoMapPreservesContract({ Title: 'Mpox - Country X', ItemDefaultUrl: '/mpox-x' }, NOW);
```
[the remark this answers is not in the document]
Comment (koala73) [reply tests/disease-outbreaks-seed.test.mjs]: Fixed in baa505c26. Extracted parsers/mappers/contentMeta/publishTransform into `scripts/_disease-outbreaks-helpers.mjs` and refactored the seeder to import them. The test file now imports the real `whoNormalizeItem`/`rssNormalizeItem`/`tghNormalizeItem`/`diseaseContentMeta`/`diseasePublishTransform` from that module — no local replicas, so a parser-shape change in the seeder would now fail tests.

Net: -265 lines across the seeder + test files, +200 in the new helpers module. 17/17 disease tests still pass; 49/49 across the full Sprint 1+2 stack.

File: tests/disease-outbreaks-seed.test.mjs [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+
+test('contentMeta picks newest and oldest from the non-synthetic set', () => {
+  const NEWEST = 1700000000000;
+  const OLDEST = 1690000000000;
+  const data = {
+    outbreaks: [
+      { _publishedAtIsSynthetic: false, _originalPublishedMs: OLDEST },
+      { _publishedAtIsSynthetic: false, _originalPublishedMs: NEWEST },
+      { _publishedAtIsSynthetic: false, _originalPublishedMs: (NEWEST + OLDEST) / 2 },
+    ],
```
[the remark this answers is not in the document]
Comment (koala73) [reply tests/disease-outbreaks-seed.test.mjs:120]: Fixed in baa505c26. `diseaseContentMeta` now accepts an optional `nowMs` parameter — production callers omit it (defaults to `Date.now()`), tests inject a fixed `FIXED_NOW = 1700000000000` so the comparison against `skewLimit` is deterministic regardless of CI scheduler timing. Also tightened the "near-future within 1h tolerance" test from +30min to +5min ahead of NOW, well clear of the 1h boundary even without the deterministic-now fix.
